src/libraries/Common/src/Interop/Windows/BCrypt/Cng.cs

```diff
@@ -70,7 +71,7 @@ public enum OpenAlgorithmProviderFlags : int
 
         public static SafeAlgorithmHandle BCryptOpenAlgorithmProvider(string pszAlgId, string pszImplementation, OpenAlgorithmProviderFlags dwFlags)
         {
-            SafeAlgorithmHandle hAlgorithm = null;
+            SafeAlgorithmHandle? hAlgorithm = null;
```

Comment:
Why does this need to be nullable?  I get that it was being assigned to null, but that assignment looks superfluous / unnecessary.